src/OpenConext/ProfileBundle/Controller/MyServicesController.php

```diff
@@ -49,30 +50,44 @@ class MyServicesController
     private $guard;
 
     /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
      * @param EngineInterface $templateEngine
      * @param AuthenticatedUserProvider $authenticatedUserProvider
      * @param SpecifiedConsentListService $specifiedConsentListService
      * @param Guard $guard
+     * @param LoggerInterface $logger
      */
     public function __construct(
         EngineInterface $templateEngine,
         AuthenticatedUserProvider $authenticatedUserProvider,
         SpecifiedConsentListService $specifiedConsentListService,
-        Guard $guard
+        Guard $guard,
+        LoggerInterface $logger
     ) {
         $this->templateEngine              = $templateEngine;
         $this->authenticatedUserProvider   = $authenticatedUserProvider;
         $this->specifiedConsentListService = $specifiedConsentListService;
         $this->guard                       = $guard;
+        $this->logger                      = $logger;
     }
 
     public function overviewAction()
     {
         $this->guard->userIsLoggedIn();
 
-        $user = $this->authenticatedUserProvider->getCurrentUser();
+        $this->logger->info('Retrieving specified consent list for the current user');
```

Comment:
This does not tell what the user has requested (what *this* controller action does), it tells what the system is doing in order to  comply with the request.